tpm.go

```diff
@@ -20,21 +20,893 @@
 package fdeutil
 
 import (
+	"bytes"
+	"crypto"
+	"crypto/rsa"
+	_ "crypto/sha256"
```

Comment:
is this "_" right?

Reply:
I have to do this because the file doesn't use any symbols from crypto/sha256 but it needs to be linked in so that crypto.SHA256.New() works (see https://golang.org/pkg/crypto/#Hash).